model/source.go

```diff
@@ -86,3 +86,17 @@ func (src *Source) ToResponse() *SourceResponse {
 		SourceTypeId:        &stid,
 	}
 }
+
+// ToInternalResponse returns only the fields that "sources-monitor-go" requires.
+func (src *Source) ToInternalResponse() *SourceInternalResponse {
+	id := strconv.FormatInt(src.ID, 10)
+	tenantId := strconv.FormatInt(src.TenantID, 10)
+
+	source := &SourceInternalResponse{
+		Id:                 &id,
+		AvailabilityStatus: &src.AvailabilityStatus.AvailabilityStatus,
+		TenantId:           &tenantId,
```

Comment:
from request on ruby app - it looks that we are exposing `Tenant.ExternalTenant` for attribute "tenant" in response .

Reply:
You are right. I tried it by setting up the Rails app on the local environment and we're returning the external tenant. I changed that.